downstream/modules/installing_ocp_prerequisites.adoc

```diff
@@ -18,11 +18,13 @@ To obtain the OpenShift Operator Metering usage data, cost management requires t
 *On your OpenShift cluster*
 
 * OpenShift Container Platform 4.3
-* Operator Metering
+* Cost Management Operator
+////
 +
-Operator Metering is not deployed out of the box in OpenShift Container Platform. See https://docs.openshift.com/container-platform/4.3/metering/metering-installing-metering.html[Installing Metering] in the OpenShift documentation for instructions to install metering from OperatorHub.
+The Cost Management Operator is not deployed out of the box in OpenShift Container Platform. See https://docs.openshift.com/container-platform/4.3/metering/metering-installing-metering.html[Installing Metering] in the OpenShift documentation for instructions to install the Operator from OperatorHub.
```

Comment:
```suggestion
The Cost Management Operator is not deployed out of the box in OpenShift Container Platform. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html/metering/installing-metering[Installing Metering] in the OpenShift documentation for instructions to install the Operator from OperatorHub.
```

If you search "Metering" on https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/
You will find it under **Manage**.

This is the index page.
https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html/metering/index